backend/api/Areas/Documents/Controllers/DocumentController.cs

```diff
@@ -0,0 +1,66 @@
+
+using MapsterMapper;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Pims.Api.Models.Concepts;
+using Pims.Api.Policies;
+using Pims.Api.Services;
+using Pims.Dal.Security;
+using Swashbuckle.AspNetCore.Annotations;
+using System.Collections.Generic;
+
+namespace Pims.Api.Areas.Notes.Controllers
```

Comment:
This looks like is in the wrong namespace

Reply:
Deleted the controller as moved the endpoint in Controllers\DocumentController.cs